<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mogs_video_widget</class>
 <widget class="QMainWindow" name="mogs_video_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>677</width>
    <height>447</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>mogs_video_widget</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QSplitter" name="splitter">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QWidget" name="layoutWidget">
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="sizeConstraint">
           <enum>QLayout::SetMaximumSize</enum>
          </property>
          <item>
           <widget class="mogs_qt_list_point" name="tableView">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>400</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="mogs_qt_list_video" name="listView_2">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>400</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="Label_selected_video">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="layoutWidget">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QGraphicsView" name="OpenCVWindow">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>320</width>
              <height>0</height>
             </size>
            </property>
            <property name="autoFillBackground">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QPushButton" name="pushButton">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>Manual editing</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_2">
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>150</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>Automatic editing</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="play_pause_button">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>play/pause</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QScrollBar" name="horizontalScrollBar">
          <property name="maximumSize">
           <size>
            <width>2000</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>677</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>menu</string>
    </property>
    <addaction name="actionInit"/>
    <addaction name="separator"/>
    <addaction name="actionNouveau_projet"/>
    <addaction name="actionOuvrir_projet"/>
    <addaction name="separator"/>
    <addaction name="actionAjouter_point"/>
    <addaction name="actionAjouter_video"/>
    <addaction name="separator"/>
    <addaction name="actionSupprimer_point"/>
    <addaction name="actionSupprimer_video"/>
    <addaction name="separator"/>
    <addaction name="actionFermer_2"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionInit">
   <property name="text">
    <string>Initialisation</string>
   </property>
  </action>
  <action name="actionAjouter_point">
   <property name="text">
    <string>Ajouter point</string>
   </property>
  </action>
  <action name="actionAjouter_video">
   <property name="text">
    <string>Ajouter video</string>
   </property>
  </action>
  <action name="actionFermer">
   <property name="text">
    <string>Fermer</string>
   </property>
  </action>
  <action name="actionSupprimer_point">
   <property name="text">
    <string>Supprimer point</string>
   </property>
  </action>
  <action name="actionSupprimer_video">
   <property name="text">
    <string>Supprimer video</string>
   </property>
  </action>
  <action name="actionFermer_2">
   <property name="text">
    <string>Fermer</string>
   </property>
  </action>
  <action name="actionNouveau_projet">
   <property name="text">
    <string>Nouveau projet</string>
   </property>
  </action>
  <action name="actionOuvrir_projet">
   <property name="text">
    <string>Ouvrir projet</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>mogs_qt_list_video</class>
   <extends>QListView</extends>
   <header>mogs_qt_list_video.h</header>
  </customwidget>
  <customwidget>
   <class>mogs_qt_list_point</class>
   <extends>QTableView</extends>
   <header>mogs_qt_list_point.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
